database.sql

```diff
@@ -1,23 +1,100 @@
--- Step 1: Insert a user
+
+
+
+
+
+CREATE TABLE IF NOT EXISTS Users(
+    UserID TEXT UNIQUE NOT NULL,
+    password CHAR(80) NOT NULL,
+    firstName CHAR(80) NOT NULL,
+    lastName CHAR(80) NOT NULL,
+    PRIMARY KEY(USERID)
+    
+);
+
+
+CREATE TABLE IF NOT EXISTS Orders (
+    OrderID TEXT UNIQUE NOT NULL,
+    UserID TEXT NOT NULL,
+    BusinessID TEXT NOT NULL,
+    ShopperID TEXT NOT NULL,
```

Comment:
This needs to be `NULL`able because an entry to this table will be made before a shopper is selected.